cask/fastlane.rb.template

```diff
@@ -11,3 +11,18 @@ cask 'fastlane' do
 
```

Comment:
Line 5 and 6 (GitHub won't let me comment on the line) might need to be changed to point to the new install location.

Reply:
@taquitos I JUST updated this file to what is actually in cask as you were commenting probably 🙃